### PR TITLE
style: Label styling/storybook touchups

### PR DESCRIPTION
**Pull request by rusackas**
### SUMMARY

This PR makes a few little tweaks to the Label component:
1) Removes a bunch of Emotion styles that weren't being used (seemingly copied from Button)
2) Adds a bunch of Emotion styles to pull the correct Theme color variables in
3) Makes it so that label colors only darken on hover if there's an onClick handler
4) Changes the cursor of the Label to 'pointer' if there's an onClick handler
5) Adds the "info" label to the Storybook
6) Breaks out the Gallery and Interactive parts of the storybook entry to different Stories for simplicity (I should do this with Button, too). This makes certain styling work better, particularly removing the `:first-child` left margins in this case.
7) Adds an "onClick action" knob to the interactive storybook entry

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF

After:
Pasted_Image_8_17_20__11_05_PM
Pasted_Image_8_17_20__11_09_PM
Pasted_Image_8_17_20__11_10_PM

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Changes UI
- [ ] Requires DB Migration.
- [ ] Confirm DB Migration upgrade and downgrade tested.
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

File: superset-frontend/src/components/Label/index.tsx [hunk 1]
```diff
@@ -34,40 +34,40 @@ export interface LabelProps {
...
+  &.label-default {
+    background-color: ${({ theme }) => theme.colors.grayscale.base};
+    &:hover {
+      background-color: ${({ theme, onClick }) =>
+        onClick ? theme.colors.grayscale.dark1 : theme.colors.grayscale.base};
     }
   }
-
-  &.secondaryLabel {
-    background-color: ${({ theme }) => theme.colors.secondary.base};
```
Comment (nytai) [inline superset-frontend/src/components/Label/index.tsx:68]: This .secondaryLabel style is being used by the card component for charts.

Comment (nytai) [reply superset-frontend/src/components/Label/index.tsx:68]: see: https://github.com/preset-io/incubator-superset/blob/master/superset-frontend/src/views/CRUD/chart/ChartList.tsx#L543

**Comment by rusackas** [comment]
Now includes `primary` and `secondary` label styles, implemented just like the main bootstrap ones, with the ability to add more as needed.
image

**Review by nytai — APPROVED** [review]
lgtm, thanks for cleaning this up and making the storybook stories rich.

**Pull request merged**